mobile/library/cc/request_method.cc

```diff
@@ -1,19 +1,20 @@
 #include "request_method.h"
 
 #include <stdexcept>
-#include <vector>
+
+#include "absl/strings/string_view.h"
 
 namespace Envoy {
 namespace Platform {
 
-static const std::pair<RequestMethod, std::string> REQUEST_METHOD_LOOKUP[]{
+static const std::pair<RequestMethod, absl::string_view> REQUEST_METHOD_LOOKUP[]{
```

Comment:
nit: as long as you're here, mind moving this into an anon namespace?

Reply:
Done